src/routes/reference/nmffilter.svx

```diff
@@ -1,12 +1,124 @@
+<a href='/reference/nmffilter/nmffilter-example.maxpat' download='nmffilter-example.maxpat'>Max example code for the sounds heard below</a>
+
+In this example, we’ll use the bases from the analysis on the [BufNMF](/overviews/bufnmf) page. These bases contain spectral templates for the (1) kick drum, (2) snare drum, and (3) hi-hat components of the original drum loop.
+
+<Audio
+src="/audio/Nicol-LoopE-M.mp3"
+label="The original drum loop used to derive the bases from a BufNMF analysis"
+/>
+
+First, let’s send the original drum loop through NMFFilter in real-time using the bases derived from the drum loop analysis.
+
+### Audio stream output based on basis #1:
+<Audio
+src="/reference/nmffilter/02_orig_through_nmffilter_1.mp3"
+label="The output audio of the original drum loop filtered by basis #1 in NMFFilter"
+/>
+
+### Audio stream output based on basis #2:
+<Audio
+src="/reference/nmffilter/02_orig_through_nmffilter_2.mp3"
+label="The output audio of the original drum loop filtered by basis #2 in NMFFilter"
+/>
+
+### Audio stream output based on basis #3:
+<Audio
+src="/reference/nmffilter/02_orig_through_nmffilter_3.mp3"
+label="The output audio of the original drum loop filtered by basis #3 in NMFFilter"
+/>
+
+This will give us very similar sounding results as BufNMF’s resynthesized buffer–the difference here is that results are produced in real-time. This means that one could send a live input of this drummer and (hopefully) get similarly successful results decomposing the drum loop into the individual drum instruments in real-time.
+
+## Audio input from a different source
+
+One could also send a different audio signal through these bases using NMFFilter. Here we’ll use an excerpt from an ensemble recording that also has kick drum, snare drum, and hi-hat.
+
+### Original ensemble recording:
+
+<Audio
+src="/audio/Tremblay-beatRemember.mp3"
+label="Source ensemble recording."
+/>
+
+### Audio stream output based on basis #1:
+
+<Audio
+src="/reference/nmffilter/03_song_through_nmffilter_1.mp3"
+label="The output audio of the ensemble recording filtered by basis #1 of the drum loop BufNMF analysis"
+/>
+
+### Audio stream output based on basis #2:
+
+<Audio
+src="/reference/nmffilter/03_song_through_nmffilter_2.mp3"
+label="The output audio of the ensemble recording filtered by basis #2 of the drum loop BufNMF analysis"
+/>
+
+### Audio stream output based on basis #3:
+
+<Audio
+src="/reference/nmffilter/03_song_through_nmffilter_3.mp3"
+label="The output audio of the ensemble recording filtered by basis #3 of the drum loop BufNMF analysis"
+/>
+
+NMFFilter has relatively well separated the hi hat, kick drum, and snare drum sounds in this song (because those are the bases from the drum loop!). Of course there are other sounds in each of these audio signals as well, as all of the sound from the original file needs to be represented somewhere.
```

Comment:
I think the paranthentical could be removed in favour of a comma.

`in this song, because those are the bases from the drum loop`